src/python/turicreate/toolkits/_model.py

```diff
@@ -513,6 +513,9 @@ def summary(self, output=None):
             return self.__class__.__name__
 
 
+    def _list_fields(self):
```

Comment:
This seems strange. You are simply calling the same function by itself? Am I missing something here. Isn't this a recursive call?

Reply:
Yeah, this is an oversight. Taking it out. Will rebase and squash to update the base commit for all other `__repr__` PRs.